tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "target": "esnext",
+        "module": "commonjs",
+        "allowJs": true,
+        "sourceMap": true,
+        "outDir": "./dist",
+        "strict": true,
```

Comment:
Codacy found an issue: [Comment not allowed](https://app.codacy.com/gh/P2P-Develop/TopDomainChecker/pullRequest?prid=6364165)